frontend/src/Pages/Herosection.css

```diff
@@ -2,14 +2,14 @@
 @tailwind components;
 @tailwind utilities;
 
-body{
+body {
     padding: 0;
     margin: 0;
-   
-   background-image: url('../assets/mixbg.png');
+    background-image: url('../assets/mixbg.png');
     background-size: cover;
     background-position: center;
     height: 100vh;
-    width: 100%;
-}
-
+    width: 100vw; 
+    overflow: hidden; 
```

Comment:
Please remove it 
Do  `overflow-x : hidden`

Reply:
ok